internal/nginx/templates/nginx-plus.tmpl

```diff
@@ -96,10 +96,10 @@ http {
 
         location  = /dashboard.html {
         }
-
-        allow 127.0.0.1;
+{{ range $value := .NginxStatusAllowCIDRs }}{{ if ne $value "" }}
```

Comment:
could you also remove the whitespace in `{{ ... }}` to be more consisten with the rest of the template? For example, `{{ range $value := .NginxStatusAllowCIDRs }}` -> `{{range $value := .NginxStatusAllowCIDRs}}`